shared/src/TlsVersions.cs

```diff
@@ -47,6 +47,11 @@ internal TlsVersions()
         private const SslProtocols AllowedProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
         private const SslProtocols PreferredProtocol = SslProtocols.Tls12;
 
+        /// <summary>
+        /// To enable certificate revocation check. Default to be true.
+        /// </summary>
+        public bool CertificateRevocationCheck { get; set; } = true;
```

Comment:
You could add a unit test to the TlsVersionsTests file to confirm this defaults to true.